packages/core/content-type-builder/admin/src/components/IconPicker/tests/index.test.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { IntlProvider } from 'react-intl';
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
I don't think fireEvent is needed as you can do all it does can be done with userEvent